src/components/Complete/index.js

```diff
@@ -1,33 +1,12 @@
 import { h, Component } from 'preact'
-import theme from '../Theme/style.css'
-import style from './style.css'
-import { trackComponent } from '../../Tracker'
+import VerificationComplete from './VerificationComplete.js'
 
 class Complete extends Component {
-  constructor (props) {
-    super(props)
-  }
-
-  componentDidMount () {
-    this.props.nextStep()
-  }
-
-  render ({message, submessage}) {
+  render (props) {
```

Comment:
This component doesn't do anything - you can just merge `VerificationComplete` into `Complete`